scripts/demo.js

```diff
@@ -0,0 +1,34 @@
+import path from "path";
+import { execSync } from "child_process";
+import fs from "fs";
+
+const demoPath = path.join(process.cwd(), "demo");
+
+const options = {
+  encoding: "utf8",
+  stdio: "inherit",
+  cwd: path.dirname(demoPath)
+};
+const yaml = path.join(process.cwd(), "cookiecutter.yaml");
+
+function removeDir(dir) {
+  if (fs.existsSync(dir)) {
+    fs.rmSync(dir, { recursive: true });
+  }
+}
+
+removeDir(demoPath);
+
+const rnCookieCutterCommand = [
+  "pipenv run cookiecutter",
+  "./dist/cookie",
+  "--checkout master",
```

Comment:
This argument shouldn't be needed given the local path argument above

Reply:
Oops an oversight. Updated with [a21faaf](https://github.com/crowdbotics/react-native-scaffold/pull/431/commits/a21faaf7ab6a0933366cf208edeba699ffb64f1d)